android/src/main/java/cl/json/social/ShareIntent.java

```diff
@@ -195,6 +195,10 @@ public static boolean hasValidKey(String key, ReadableMap options) {
 
     protected abstract String getPackage();
 
+    protected String getComponentClass() {
+        return null;
```

Comment:
Is this right to return null here?

Reply:
@jgcmarins yeah it's a default connected to the overrides in subclass overrides in response to previous review